cmd/availabilityZone.go

```diff
@@ -24,7 +24,7 @@ var availabilityZoneListCmd = &cobra.Command{
 	Use:     "list",
 	Short:   "This command shows a list of availability zones.",
 	Long:    "This command shows a list of availability zones. Availability zones are changes according to cloud provider, region,instance type and needed count",
```

Comment:
I'd rephrase this as well to something like:
Availability zones depend on the cloud provider, region, instance type and count

(currently it's less clear and has minor typos)